.github/actions/update-rapidpro-workspace-settings/README.md

```diff
@@ -0,0 +1,60 @@
+# update-rapidpro-workspace-settings Shared GitHub Action
```

Comment:
```suggestion
# variable-rapidpro-workspace Shared GitHub Action
```

Nit - maybe rename to be about the problem that is being solved rather than the action that the tool is doing?

Reply:
Agreed.